packages/slate-editor/src/components/Menu/Dropdown.tsx

```diff
@@ -44,30 +45,36 @@ export function Dropdown<Value extends string = string>({
     }
 
     return (
-        <BootstrapDropdown
-            {...props}
+        <Listbox
             className={classNames(styles.Dropdown, className)}
-            onSelect={handleSelect}
-            open={open}
-            onToggle={(isOpen, _, meta) => meta.source !== 'rootClose' && setOpen(isOpen)}
+            as="div"
+            {...props}
+            value={selected}
+            onChange={handleSelect}
```

Comment:
I've noticed at least two regressions (in Firefox):

1. There is unwanted "focused" state ring around menu elements when I click around
2. There is a horizontal scrollbar appearing in the formatting dropdown menu

[Screencast from 2023-05-17 15-38-51.webm](https://github.com/prezly/slate/assets/370680/2bcc0798-50d6-45f9-9672-9cabb801d5ab)

Reply:
> I was testing with the attached editor in Prezly

Yeah, that's one of the reasons I've created CARE-1580 -- some problems are not noticeable in the Prezly application. Like implicit dependency on bootstrap.css.

So testing it outside the app seems to be a good idea too :)